alvin/README.md

```diff
@@ -0,0 +1,12 @@
+A wordle
+
+My attempt at replicating wordle to implement the lessons learnt over the last 4 weeks.
+Focus on maximising use of DOM manipulation through JavaScript, and applying styling techniques in CSS to improve page aesthetics.
+
+Future improvements:
+* Adding functionality to onscreen keyboard.
+* Improving responsiveness across webpage sizes and device types.
+* Addition of local storage
+* Addition of score board or scoring method
+* General simplification of code 
+
```

Comment:
nice and clear explanation. I'd also add a link to a live demo hosted on github pages.

Reply:
Also, I'd include a screenshot of the game in the README.